backend/clubs/views.py

```diff
@@ -2539,7 +2538,9 @@ class QuestionAnswerViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         club_code = self.kwargs["club_code"]
-        questions = QuestionAnswer.objects.filter(club__code=club_code)
+        questions = QuestionAnswer.objects.filter(
+            club__code=club_code, club__archived=False
+        )
```

Comment:
@ezwang would
```QuestionAnswer.objects.filter(club__code=club_code).select_related("archived").filter(club__archived=False)```
be better here? I tested with DjDT and it looks like this isn't an N+1 issue, but the Django ORM has tricked me before so figured I should double check

Reply:
You are correct, this would not be better here. Django can automatically optimize filter conditions, and adding a filter condition shouldn't ever cause a N+1 issue.